### ISSUE TITLE
(BUG ) ICC loot template RequeridQuest Rotface's and Festergut's Acidic Blood and Shadowfrost Shard 

### ISSUE THREAD

**Issue by Fireevil** [azerothcore/azerothcore-wotlk#16653]
### Current Behaviour

-only a player can see these items although more people have the mission

-the rest of the raid cannot see these articles even though they have the mission

![Screenshot_12](https://github.com/azerothcore/azerothcore-wotlk/assets/88008285/503daaac-a7e7-4392-bd7c-3861047918c4)
![Screenshot_13](https://github.com/azerothcore/azerothcore-wotlk/assets/88008285/7212cc09-268d-461f-bed2-0daaeb77731f)

### Expected Blizzlike Behaviour

all people who have the quest should see the item in the loot

### Steps to reproduce the problem

1- 3 player with mission https://www.wowhead.com/wotlk/es/quest=24548/el-trono-astillado
2- loot in free mode or ML
2-kill any boss Example my test was in : Rotface's , Festergut's and valanar
3- only one player can see the loot : Rotface's and Festergut's Acidic Blood and Shadowfrost Shard 

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/cd01453b6c1d54aea5f447c43213d36df7db09f0

Last Rev

### Operating system

Windows 10

**Comment by Tony931023** [comment]
confirmed this is happening, sometimes a single player with the mission does not see the item either, or even only lets a player who has completed the mission see it or take it

**Comment by Fireevil** [comment]
Yes confirmed too in more ítems of quest

**Comment by Fireevil** [comment]
any fix??????

**Comment by smellbee** [comment]
confirmed，also Rotface's Acidic Blood and Festergut's Acidic Blood have the same issue

**Comment by EricksOliveira** [comment]
Recently the problem started after 2 updates.
https://github.com/azerothcore/azerothcore-wotlk/commit/784c5e941605b5e2b8ece02183bf9253bca51012
https://github.com/azerothcore/azerothcore-wotlk/commit/7f9cdda17bf1acc026072ff9993fefa5a2ffdae7

**Comment by Voidx** [comment]
Is this issue still ongoing? I'm receiving reports about loot problems.

**Comment by EricksOliveira** [comment]
Yes. I sent a PR, but it needs testing. I tested it on a private server and it worked fine for me, no issues.

**Comment by Voidx** [comment]
@EricksOliveira i can test it too
